Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs

```diff
@@ -4,7 +4,116 @@ internal class Program
+                    break;
+                case "4":
+                    Console.Clear();
+                    DivisionGame();
+                    break;
+                default:
+                    Console.Clear();
+                    Console.WriteLine("Out of bounds. Please try again.");
+                    Environment.Exit(0);
+                    break;
+            }
+        }
+
+        //Addition
+        public static void AdditionGame()
+        {
+            Console.WriteLine("\t\t\t\t\t\tWelcome to the Addition Game\n\n\n");
+            Console.WriteLine("Please input the 2 numbers: ");
+            Console.Write("First Number: ");
+            int num1 = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Second Number: ");
+            int num2 = Convert.ToInt32(Console.ReadLine());
+            int sum = num1 + num2;
+            Console.WriteLine($"The sum of the numbers, {num1} and {num2} is: {sum}");
+            Console.WriteLine("Please enter a key to continue. . .");
+            Console.ReadLine();
+        }
+
+        //Subtraction
+        public static void SubtractionGame()
+        {
+            Console.WriteLine("\t\t\t\t\t\tWelcome to the Subtraction Game\n\n\n");
+            Console.WriteLine("Please input the 2 numbers: ");
+            Console.Write("First Number: ");
+            int num1 = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Second Number: ");
+            int num2 = Convert.ToInt32(Console.ReadLine());
+            int difference = num1 - num2;
+            Console.WriteLine($"The difference of the numbers, {num1} and {num2} is: {difference}");
+            Console.WriteLine("Please enter a key to continue. . .");
+            Console.ReadLine();
+        }
+
+        //Multiplication
+        public static void MultiplicationGame()
+        {
+            Console.WriteLine("\t\t\t\t\t\tWelcome to the Multiplication Game\n\n\n");
+            Console.WriteLine("Please input the 2 numbers: ");
+            Console.Write("First Number: ");
+            int num1 = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Second Number: ");
+            int num2 = Convert.ToInt32(Console.ReadLine());
+            int product = num1 * num2;
+            Console.WriteLine($"The product of the numbers, {num1} and {num2} is: {product}");
+            Console.WriteLine("Please enter a key to continue. . .");
+            Console.ReadLine();
+        }
+
+        //Division
+        public static void DivisionGame()
+        {
+            Console.WriteLine("\t\t\t\t\t\tWelcome to the Division Game\n\n\n");
+            Console.WriteLine("Please input the 2 numbers: ");
+            Console.Write("First Number: ");
+            int num1 = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Second Number: ");
+            int num2 = Convert.ToInt32(Console.ReadLine());
+            int quotient = num1 / num2;
```

Comment:
This currently does not protect from the user entering 0 as the second number, which results in an unhandled exception. The program should ensure that either the user cannot enter 0 for the second number, or it should display NaN or something similar.